README.md

```diff
@@ -43,65 +43,67 @@ GPU: NVIDIA Corporation GV100GL [V100 PCIe 16GB]
+If running the application locally,  `PUBLIC_IP` can be set to `localhost`.
 
-2. Run `launch.sh`.  
+```
+export PUBLIC_IP=<hostname>
+```
+
+2. Please ensure that the following three ports are free and available: `50051`, `50059` and `82`
 
-3. By default, step 2 launches the containers in 'cpu' mode. To launch with gpu support, run launch.sh with `-m gpu` argument as follows `launch.sh -m gpu`. 
+2. Run `launch.sh`  
+
+3. By default, step 2 launches the containers in 'cpu' mode. To launch with gpu support, run launch.sh with `-m gpu` argument as follows `launch.sh -m gpu`
 
 🚨 **Note**: This process will take a while to complete as it will download necessary docker images and bring up services.
 
 
 <h3>⚙️ Configuration </h3>
 
-1. Run `docker ps` to verify all three containers (primeqa-ui, primqa-orchestrator and primeqa-service) are running.
+1. Run `docker ps` to verify that all the three containers (primeqa-ui, primqa-orchestrator and primeqa-service) are running.
 
-2. You will need to configure few additional settings before first use. These setting are intentionally left blank for security purposes, 
+2. You will need to configure a few additional settings before first use. These setting are intentionally left blank for security purposes. 
 
-3. Open browser of choice (Mozilla Firefox/Google chorme) and visit "http://`{PUBLIC_IP}`:50059/docs". This url shows available orchestrator APIs.
+3. Open your browser of choice (Mozilla Firefox/Google Chorme) and visit "http://`{PUBLIC_IP}`:50059/docs". This url shows the available orchestrator APIs.
 
-4. Click on [PATCH] `/settings`. Once expanded, click on `Try it out` button and copy-paste following content in the request body section.
-    
-    a. For IBM® Watson Discovery based retriever, update `Watson Discovery` releated section in `retrievers`
-    ```json
-        "Watson Discovery": {
-            "service_endpoint": "<IBM® Watson Discovery Cloud/CP4D Instance Endpoint>",
-            "service_token": "<Bearer token (If using IBM® Watson Discovery CP4D Instance)>",
-            "service_api_key": "<API key (If using IBM® Watson Discovery Cloud instance)>",
-            "service_project_id": "<IBM® Watson Discovery Project ID>"
-        }
-    ```
+4. Click on [PATCH] `/settings`. Once expanded, click on the `Try it out` button and copy-paste the Retriever and Reader settings that you would like to use from the examples below. 
 
-    b. For PrimeQA based retrievers, update `PrimeQA` related section in `retrievers` as follows
-    ```json
-        "PrimeQA": {
-            "service_endpoint": "primeqa:50051"
-        }
-    ```
+    a. To use the IBM® Watson Discovery Cloud/CP4D retriever and PrimeQA reader, first configure a Watson Discovery instance using the instructions [here](https://cloud.ibm.com/catalog/services/watson-discovery).
```

Comment:
Instruction are how to setup up  a `cloud` instance of WD. I would take `CP4D` mention from step.